VideoLocker/src/main/java/org/edx/mobile/util/SecurityUtil.java

```diff
@@ -13,20 +12,31 @@
 import java.io.File;
 import java.util.Collections;
 
-public abstract class SecurityUtil {
+/**
+ * Utility class that deals with the security of a user's personal information data.
+ */
+public class SecurityUtil {
     private static final Logger logger = new Logger(SecurityUtil.class);
 
+    private SecurityUtil() {
+    }
+
+    /**
+     * Clears the app's data directory, external storage directory and shared preferences with the
```

Comment:
> comma before "and"

there's never a comma before "and" in these situations e.g.

- I'll do this, this and that. (correct)
- I'll do this, this, and that. (incorrect)